tests/unit/test_internal_utils.py

```diff
@@ -430,18 +431,34 @@ def test_convert_batch_dataset_to_batch_list_invalid_type_sparse(_mock: MagicMoc
         convert_batch_dataset_to_batch_list(update_data)
 
 
+data_filter_all = ComponentAttributeFilterOptions.ALL
+data_filter_relevant = ComponentAttributeFilterOptions.RELEVANT
```

Comment:
* why do you want this? if the name is too long, you can also use the `from ... import ... as ...` syntax
* it's good practice to capitalize global variables (cfr. PEP 8 (https://peps.python.org/pep-0008/#constants))

```suggestion
DATA_FILTER_ALL = ComponentAttributeFilterOptions.ALL
DATA_FILTER_RELEVANT = ComponentAttributeFilterOptions.RELEVANT
```

Reply:
Addressed.